lighthouse-viewer/app/src/github-api.js

```diff
@@ -116,7 +116,9 @@ class GithubApi {
             const filename = Object.keys(json.files)
                 .find(filename => filename.endsWith(GithubApi.LH_JSON_EXT));
             if (!filename) {
-              throw new Error(`Failed to find a Lighthouse report (*${GithubApi.LH_JSON_EXT}) in gist ${id}`);
+              throw new Error(
```

Comment:
This was to fix a conflict that was introduced from a different PR somewhere along the road.